frappe/utils/change_log.py

```diff
@@ -155,11 +155,11 @@ def check_for_update():
 		for update_type in updates:
 			if github_version.__dict__[update_type] > instance_version.__dict__[update_type]:
 				updates[update_type].append(frappe._dict(
-					current_version   = str(instance_version),
+					current_version	  = str(instance_version),
 					available_version = str(github_version),
-					org_name          = org_name,
-					app_name          = app,
-					title             = apps[app]['title'],
+					org_name		  = org_name,
+					app_name		  = app,
+					title			  = apps[app]['title'],
```

Comment:
Lets remove the spaces before `=` and keep just one.

Reply:
There should be no space at all though right? Since these are kwargs

Reply:
Oops, didn't realise that 😅